spring-pulsar/src/main/java/org/springframework/pulsar/core/PulsarAdministration.java

```diff
@@ -61,12 +64,28 @@
 	 * @param adminConfig the {@link PulsarAdmin} configuration
 	 */
 	public PulsarAdministration(Map<String, Object> adminConfig) {
-		this.adminBuilder = PulsarAdmin.builder().loadConf(adminConfig);
+
+		this.adminBuilder = PulsarAdmin.builder();
+
+		Map<String, Object> conf = new HashMap<>(adminConfig);
```

Comment:
Wdyt about abstracting this via a utility method like we do in the other client builders?

Reply:
done